shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1377,8 +1377,16 @@ void WebContents::HandleNewRenderFrame(
     std::string color_name;
     if (web_preferences->GetPreference(options::kBackgroundColor,
                                        &color_name)) {
-      rwhv->SetBackgroundColor(ParseHexColor(color_name));
+      web_contents()->SetPageBaseBackgroundColor(ParseHexColor(color_name));
     } else {
+      web_contents()->SetPageBaseBackgroundColor(absl::nullopt);
+    }
+
+    // When a page base background color is set, transparency needs to be
+    // explicitly set by calling
+    // RenderWidgetHostOwnerDelegate::SetBackgroundOpaque(false).
+    // RenderWidgetHostViewBase::SetBackgroundColor() will do this for us.
+    if (web_preferences->IsEnabled(options::kTransparent)) {
```

Comment:
This is undocumented.
Ref: https://github.com/electron/electron/issues/16212#issuecomment-962457382

Reply:
@jkleinsc @ChALkeR just a reminder to add this to the docs :)